### ISSUE TITLE
Line-breaking issues

### ISSUE THREAD

**Issue by sejr** [mattboldt/typed.js#60]
When I am typing a sentence that goes outside of the parent container, Typed.JS stretches my page horizontally. I see that the project's homepage doesn't have this issue, but I can't figure out how to implement it myself.

**Comment by mattboldt** [comment]
You need to tweak your overflow and text-wrapping settings. If the text wrapping isn't set right it'll keep going and probably expand the parent div like what you're seeing.

**Comment by highspeedtree** [comment]
Hey, I'm having issues with this as well. The type will stop at a line break within the parent div and the cursor moves to the next line but the typing doesn't continue...?

**Comment by topherhunt** [comment]
+1 same issue here. The typing just stops, cursor blinking on the next row, until the entire quote has completed typing (invisibly) and then the whole thing appears. See [topherhunt.github.io](http://topherhunt.github.io) for my use case.

I can only reproduce it with certain strings, not others, and depending on the container width, but it becomes much easier to reproduce when I set the container font to monospace; I think it has something to do with whether or not the final character before the line break is a whitespace.

**Comment by topherhunt** [comment]
I've managed to work around this. I figured out that this stopping glitch only occurs when the cursor is shown; if you disable the cursor, the issue disappears. And if you hide the cursor (at any point) via Jquery, the text is displayed as it should. So if do the cursor-blinky thing via JS instead of CSS, this simulates the cursor blink but also serves the double job of periodically refreshing the div in case the text gets "stuck":

    function jiggle_cursor(){
      $('.typed-cursor').toggle();
      setTimeout(function(){ jiggle_cursor(); }, 600);
    }

    jiggle_cursor();
